### DISCUSSION TITLE
Crackling noise while RX using PHONES connector

### DISCUSSION THREAD

**Discussion by rvolchkov** [ExpertSDR3/ExpertSDR3-BUG-TRACKER#118] [General]
If you use Windows OS you may experience crackling noise in RX mode when you listen to the air using the PHONES connector on the TRX. This issue is related to the particularities of Windows and network drivers. There are a couple of potential solutions that may help to solve or minimize this effect.
1. Unzip and launch this file to improve data exchange between PC and SDR. [improve udp.zip](https://github.com/ExpertSDR3/ExpertSDR3-BUG-TRACKER/files/7774283/improve.udp.zip)
it'll add two new entries to your Windows registry: "NetworkThrottlingIndex", "FastSendDatagramThreshold".

2. [Change Network Adapter Priorities](https://www.ghacks.net/2016/12/02/change-network-adapter-priorities-in-windows-10/).

3. Use a USB-LAN adapter instead of the integrated net card in the motherboard or notebook.

4. Make sure to set High-Performance mode in your video card settings.
It's always good to have a discrete video card, and not an integrated one. 

5. In Windows 10 under the **sound control panel**, go to your **playback tab** and select your default playback device (usually your speakers) then select **properties**. Now click on the **advanced tab** and under "Exclusive Mode" **UNCHECK _"Allow applications to take exclusive control of this device"_**.

6. You can add ExpertSDR3 to a [gaming mode](https://gadgetstouse.com/blog/2021/07/14/enable-gaming-mode-on-windows-11/#:~:text=The%20Gaming%20Mode%20was%20first%20introduced%20with%20the,game.%20What%20Does%20the%20Windows%20Gaming%20Mode%20Do%3F), which also gives it a higher priority and disables a few background things.

In some cases, none of these solutions may help you, if so please let us know your PC configuration:
- TRX model and firmware version
- ExpertSDR version
- CPU
- GPU, driver version (use the latest video drivers). 
- RAM
- Windows version + build

If you've found another solution to this issue, please share it with the community.

**Comment by N8SDR007** [comment]
Also note for windows 10 and 11 you can add EESDR to your gaming mode, which also gives it a higher priority and disables a few background things - Between the above which I have been doing for years and this in windows 11 it is very seldom I have an audio issue. 
Another this is to in **windows 10 /11 uncheck the audio device "Exclusive control feature" in your sound settings** 

https://gadgetstouse.com/blog/2021/07/14/enable-gaming-mode-on-windows-11/#:~:text=The%20Gaming%20Mode%20was%20first%20introduced%20with%20the,game.%20What%20Does%20the%20Windows%20Gaming%20Mode%20Do%3F

https://www.sweetwater.com/sweetcare/articles/how-to-disable-exclusive-mode-in-windows-sound-settings/
The above is helpful if you have a couple items using sound settings example and very bad for this Chrome and the New Edge browser!

**Comment by hb9bun** [comment]
Just to be clear. The crackling happens using the audio from the front PHONES jack of the SunSDR2 DX. Using the soundcard of the PC there is no crackling (just horrible keyer CW signal timing in the monitor audio signal). How does ESDR3 with any sample rate (and ESDR2 with Sample Rate at 312 kHz) cause the direct PHONES jack audio to crackle?

**Reply by M0mws** [reply]
Could this be a hardware issue. Have you tried another plug/headphones. 
Tried spraying plug with switch cleaner then plugging in and out a few times?

**Comment by hb9bun** [comment]
TRX model and firmware version
SunSDR2 DX, 
ExpertSDR version
ESDR3 0.10.0 alpha
Serial: EED06521900148
Firmware: 72.8
PCB revision: 11

CPU
Intel(R) Core(TM) i7-3770K CPU @ 3.50GHz   3.50 GHz

GPU, driver version (use the latest video drivers).
GeForce GTX 670
Driver: 456.71

RAM
16 GB

Windows version + build
Windows 10 Pro, 64 Bit
Version: 21H1
System Build: 19043.1348
Windows System Experience Pack 120.221.3920.0

**Reply by R1BLH** [reply]
> ESDR3 0.10.0 alpha
> Serial: EED06521900148
> Firmware: 72.8

I have a strange question. Why does the transceiver have firmware 72.8, which is for ESDR2, and not the one for ESDR3? First of all you should update the firmware.

**Comment by hb9bun** [comment]
Thanks to hints from other users I found a post by N8SDR in the forum that helped to get rid of the crackling: https://eesdr.com/en/forum-en/search?query=exclusive%20control&childforums=1

**"In Windows 10 under the **sound control Panel**
go to your **playback tab**- and 
select your **default playback device** (usually your speakers)then 
**select properties**
Now click on the advanced tab and 
under "**Exclusive Mode" UNCHECK "allow applications to take exclusive control of this device"****

I did that for all my audio devices and only after rebooting the PC the crackling was gone!

It seems that Windows sets these settings by default. So when you have changes or updates to sound cards/drivers they may be set again. 

73 , Peter - HB9BUN

**Comment by hb9bun** [comment]
Over the weekend I worked with another ham suffering from Crackling. In the end we found that he was using 4 monitors, 2 on an NVIDIA card, 2 on the onboard Intel card. Witout the onboard card use the crackling stopped. Actually the NVIDIA card support 4 logical monitors, so it is more about not using the onboard cards in parallel. Just for your information.

**Comment by SV1RXI** [comment]
i suffer too from this ,i try UNCHECK "Allow applications to take exclusive control of this device". and see if this continue...

**Comment by HA1AH** [comment]
I also had this issue with ExpertSDR2. The following solved almost 100% the problem:
https://ttcshelbyville.wordpress.com/2017/10/14/network-throttling-index/

**Comment by SV1RXI** [comment]
mine is solved ,thanks
